port_ocean/ocean.py

```diff
@@ -88,7 +97,11 @@ def execute_resync_all() -> None:
                 seconds=interval * 60,
                 # Not running the resync immediately because the event listener should run resync on startup
                 wait_first=True,
-            )(lambda: threading.Thread(target=execute_resync_all).start())
+            )(
+                lambda: threading.Thread(
+                    target=asyncio.run(execute_resync_all())
+                ).start()
+            )
```

Comment:
Try to implement with asyncio task instead

Reply:
But tasks needs to be awaited which is not compatible with the target type `Callable`